packages/page-council/src/Overview/Candidate.tsx

```diff
@@ -5,26 +5,32 @@
 import { AccountId, Balance } from '@polkadot/types/interfaces';
 
 import React from 'react';
+import styled from 'styled-components';
 import { AddressSmall } from '@polkadot/react-components';
 import { FormatBalance } from '@polkadot/react-query';
 
 import { useTranslation } from '../translate';
 import Voters from './Voters';
 
 interface Props {
+  className?: string;
   address: AccountId;
   balance?: Balance;
+  isPrime?: boolean;
   voters?: AccountId[];
 }
 
-export default function Candidate ({ address, balance, voters }: Props): React.ReactElement<Props> {
+function Candidate ({ className, address, balance, isPrime, voters }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
 
   return (
-    <tr>
+    <tr className={`${className} ${isPrime ? 'council--isPrime' : ''}`}>
       <td className='top'>
         <AddressSmall value={address} />
       </td>
+      <td className='top council--prime'>
+        {isPrime && t('prime voter')}
```

Comment:
Maybe just a single tick (look at society for the head. May be a crown instead of a tick... society bugs me a bit, bit the single badge is cool and doesn't take up much space)

Reply:
As for not taking up much space - in neither app is space a problem, so the text is _actually_ ok and probably better since it is very empty. Just need alignment.